### ISSUE TITLE
parse_lib_version is broken on new versions of FFmpeg

### ISSUE THREAD

**Issue by anatoly-romanov** [OpenShot/libopenshot#947]
Newer FFmpeg releases have their version information headers split into version_major.h and version.h. This causes the parse_lib_version macro from cmake/Modules/FindFFmpeg.cmake to parse them incorrectly. For example FFMPEG_AVFORMAT_VERSION is parsed as 20.100 instead of 60.20.100. This leads to various errors when trying to build libopenshot.

**Comment by marillat** [comment]
Also broken in latest ffmpeg 7.0

**Comment by jonoomph** [comment]
Yes, thanks! OpenShot needs some major work to integrate the newest versions of FFmpeg. This is on our roadmap, and hopefully we can get to it soon.
